class.jetpack-gutenberg.php

```diff
@@ -146,7 +145,13 @@ static function add_block( $type, $args, $availability ) {
 		self::register( $type, $args, $availability );
 	}
 
-	static function load( $request = null ) {
+	static function load( $response = null, $handler = null, $request = null ) {
+		$is_availability_endpoint_beta = ! is_null( $request ) && $request->get_param( 'beta' ) && wp_endswith( $request->get_route(), 'gutenberg/available-extensions' );
+
+		if ( apply_filters( 'jetpack_load_beta_blocks', $is_availability_endpoint_beta ) ) {
```

Comment:
Could you add a docblock here to please our parser (and me :) )?

Reply:
:done: